src/Chapter1.hs

```diff
@@ -553,8 +555,12 @@ value after "=" where the condition is true.
 
 Casual reminder about adding top-level type signatures for all functions :)
 -}
-
-mid x y z = error "mid: not implemented!"
+mid :: Integer -> Integer -> Integer -> Integer 
+mid x y z =
+  let maximum = max x (max y z)
+      minimum = min x (min y z)
```

Comment:
That's a very clever implementation! :brain: 

Note that `maximum` and `minimum` are names of the standard functions. Haskell allows name shadowing, that's why it works. Though, in practice, it's usually better to avoid name clashes with other functions cause it could be confusing 🙂